hack/test-integration.env

```diff
@@ -1,7 +1,7 @@
 # reduce flakiness in contended pipelines
 export KUBEBUILDER_CONTROLPLANE_START_TIMEOUT=2m
-export GOMEGA_DEFAULT_EVENTUALLY_TIMEOUT=5s
-export GOMEGA_DEFAULT_EVENTUALLY_POLLING_INTERVAL=200ms
+export GOMEGA_DEFAULT_EVENTUALLY_TIMEOUT=10s
```

Comment:
Please follow https://github.com/gardener/gardener/pull/8116#discussion_r1272081869 and revert this commit.